2017/opmodes/MochaSingleDriverTeleop.java

```diff
@@ -20,6 +21,7 @@
 import team25core.RobotEvent;
 
 @TeleOp(name="Baby Mocha", group = "5218")
+@Disabled
```

Comment:
Ok, we are apparently going to have to figure out how to bifurcate teams 25 and 5218 from an opmode perspective.    It's not very friendly to disable your sister team's opmode.   If you revert this change, I'll merge.  And over the next few weeks we will work on pulling the core infrastructure into an independent repository and team opmodes into their own repositories.  e.g.  We'll soon have FTC-Team-5218.  Until then, please do not modify your sister team's opmodes and that includes the registration annotations.